packages/tests/esm/jest.config.ts

```diff
@@ -2,10 +2,13 @@ import type { Config } from "@jest/types";
 import { createDefaultEsmPreset } from "ts-jest";
 
 export default {
-  ...createDefaultEsmPreset({ isolatedModules: true }),
+  ...createDefaultEsmPreset(),
```

Comment:
This setting trigger this warning:

```
ts-jest[config] (WARN)
    The "ts-jest" config option "isolatedModules" is deprecated and will be removed in v30.0.0. Please use "isolatedModules: true" in /Users/blimmer/code/homebound/joist-orm/packages/tests/esm/tsconfig.json instead, see https://www.typescriptlang.org/tsconfig/#isolatedModules
```

So, I've added `isolatedModules` to `tsconfig.json`